command/registry.go

```diff
@@ -55,6 +57,19 @@ func setupRegistryForPackerConfig(pc *hcl2template.PackerConfig) hcl.Diagnostics
 		}
 	}
 
+	if env.IsPAREnabled() {
```

Comment:
I'm finding something off here. `IsPAREnabled` already checks for the `HCP_PACKER_REGISTRY` so there's a duplicated logic in here. I think it will be more explicit if that is extracted to a specific method `isHCPPackerDisabled`.  This way we can just call this check before everything else, including the check of the `hcp_packer_registry` block.

If `!isHCPPackerDisabled && !hasHCPPackerRegistryBlock` then we check if HCP Packer is enabled through environment variables. Does it make sense? I'm using different names for the variables to make it easier to explain my thought. 

I think I'm also in favor of renaming PAR to HCPPacker at this point.

Reply:
This is a good call out yeah, I had previously renamed `IsPAREnabled` to `IsPARDisabled` and did that check there.
I can rework this, I'm not super satisfied with the current state of the code too, that double check I added as one of my HPATS tests failed for good reason.
I'll come back with a reroll today

Reply:
+1 on the PAR rename

Reply:
An good catch. I didn’t think to check IsPAREnabled logic. I assumed it was just the HPC_CLIENT_ and BUCKET vars. Good catch @sylviamoss 

+1 on rename.

Reply:
Good thing we have a consensus on the rename; I just pushed a reroll for this :)
Feel free to review the latest version